lightning/src/ln/channel.rs

```diff
@@ -1383,6 +1383,8 @@ pub(super) struct ChannelContext<SP: Deref> where SP::Target: SignerProvider {
 	/// The transaction which funds this channel. Note that for manually-funded channels (i.e.,
 	/// is_manual_broadcast is true) this will be a dummy empty transaction.
 	funding_transaction: Option<Transaction>,
+	/// Rememeber whether the funding transaction has been (re)broadcast (legacy)
+	funding_transaction_rebroadcast_flag: bool,
```

Comment:
Would it be equivalent to use `is_funding_broadcast` instead of adding the bool?